Scripts - General/FormatCopy/FormatCopy.pas

```diff
@@ -1066,7 +1113,7 @@ procedure FormatCopyPCB(Doc : IDocument);
                     else
                         Prompt := SourcePrim.ObjectIdString + '. Copy layer ' + Board.LayerName(SourcePrim.Layer) + ' info ?  ';
 
-                    boolLoc := MessageDlg(Prompt, mtConfirmation, mbYesNoCancel, 0);
+                    boolLoc := MessageDlg(Prompt, mtCustom, mbYesNoCancel, 0);

```

Comment:
Can use ```ConfirmNoYesCancel(Const S : String) : Integer;``` for silent dialog that matches UI theme. 
Returns 6 for Yes, 7 for No, and 2 for Cancel. (Why not just use same vals as mbYes, mbNo, mbCancel? only Altium knows)